File: ytrue-admin-server-java/ytrue-bean/src/main/java/com/ytrue/bean/resp/system/SysDeptIdResp.java
package com.ytrue.bean.resp.system;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

import java.io.Serial;
import java.io.Serializable;

@Data
public class SysDeptIdResp implements Serializable {

    @Serial
    private static final long serialVersionUID = 1L;


    @Schema(description = "唯一标识")
    private Long id;

    @Schema(description = "上级部门")
    private Long pid;


    @Schema(description = "名称")
    private String deptName;

    @Schema(description = "负责人")
    private String leader;

    @Schema(description = "联系电话")
    private String phone;

    @Schema(description = "邮箱")
    private String email;

    @Schema(description = "是否启用")
    private Boolean status;

    @Schema(description = "排序")
    private Integer deptSort;
}

File: ytrue-admin-server-csharp/Ytrue.Models/Resps/System/SysDeptIdResp.cs
namespace Ytrue.Models.Resps.System;

public class SysDeptIdResp
{
    
    /// <summary>
    /// ID
    /// </summary>
    public long Id { get; set; }
    
    /// <summary>
    /// 上级部门
    /// </summary>
    public long Pid { get; set; }
 

    /// <summary>
    /// 名称
    /// </summary>
    public string DeptName { get; set; }

    /// <summary>
    /// 排序
    /// </summary>
    public int DeptSort { get; set; }

    /// <summary>
    /// 负责人
    /// </summary>
    public string Leader { get; set; }

    /// <summary>
    /// 邮箱
    /// </summary>
    public string Email { get; set; }

    /// <summary>
    /// 联系电话
    /// </summary>
    public string Phone { get; set; }

    /// <summary>
    /// 状态:1=启用,0=禁用
    /// </summary>
    public bool Status { get; set; }
}